vlayer/ethereum/circuits/lib/src/slot_test.nr

```diff
@@ -36,6 +36,23 @@ mod mapping {
     }
 }
 
+mod mapping_by_uint {
+    use crate::slot::mapping_by_uint;
+
+    #[test]
+    fn uint_mapping() {
+        let slot = [
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10
+        ];
+        let uint = 0;
+        assert_eq(
+            [
```

Comment:
Let's change the order of args. It's a convention to do `assert_eq(tested_f(x), expected)` and not `assert_eq(expected, tested_f(x))`;

Reply:
Good point, fixed